prediction_market_agent/agents/safe_guard_agent/safe_api_models/detailed_transaction_info.py

```diff
@@ -12,69 +12,28 @@
 )
 
 
-class Sender(BaseModel):
```

Comment:
Just noticed that these types were generated as duplicities, so removing them and using just one of them